### DISCUSSION TITLE
discord plugin dont work

### DISCUSSION THREAD

**Discussion by fefucho** [pear-devs/pear-desktop#132] [Q&A]
i wanted to know what i have to do to make work the discord plugin, I alredy enable it and it doesnt work (I tried discord and discordptb)

**Comment by fefucho** [comment]
![image](https://user-images.githubusercontent.com/77519901/104795400-b47c1200-578c-11eb-98b0-bc63d2f68642.png)

i found that if u disable the custom status works, i dont know why

**Reply by semvis123** [reply]
did you restart it after enabling the plugin?

**Reply by Tsuna77** [reply]
I found that discord must be started before the app, maybe attaching is only working at launch.

**Comment by Araxeus** [comment]
I've discovered that the custom status doesn't show in the normal spot
But if you go to any discord server and look at the list of participants, you will see yourself playing youtube-music with the current song
